ContextDesktop/src/main/java/com/github/migi_1/Context/model/Environment.java

```diff
@@ -101,8 +104,11 @@ public void initialize(AppStateManager stateManager, Application app) {
     @Override
     public void update(float tpf) {
         super.update(tpf);
-        testPlatform.move(-PLATFORM_SPEED, 0, 0);
-        testCommander.move(-PLATFORM_SPEED, 0, 0);
+        float zAxis = (float) (steering*(Math.sqrt(2.f)/2.f));
```

Comment:
CheckStyle:
* should be preceded and followed by a whitespace.
/ should be preceded and followed by a whitespace.

I also think the unnecessary brackets around `steering*(Math.sqrt(2.f)/2.f)` are not needed

2.f Feels like a magic number